bower.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.11.1",
   "main": "dist/typeahead.bundle.js",
   "dependencies": {
+    "jasmine-jquery": "~2.1.1",
```

Comment:
I believe this is an error, since jasmine-jquery is put in devDependencies section.

Will fix